### DISCUSSION TITLE
parameterising new molecules

### DISCUSSION THREAD

**Discussion by katealston** [Martini-Force-Field-Initiative/MartiniDiscussions#19] [Other molecules]
I have used bartender to parameterise new molecules which I haven't been able find published etc, using bead mappings I have determined reasonable (using the martini bead bible). They are pretty small (about 20 atoms). Is it reasonable to take these as acceptable models when publishing. Is there testing I need to show in a paper before using them in other simulations? 
Thanks for all your help in advance!

**Comment by ricalessandri** [comment]
At a minimum validate the model by computing the octanol-water transfer free energy and compare it to the experimental one (logP). If you have access to more experimental transfer free energies (water/hexadecane, water/choloform), even better. A second layer of validation is to look at the mass density and then at the SASA/ molecular volume. See e.g. the small molecule paper  https://onlinelibrary.wiley.com/doi/full/10.1002/adts.202100391.
